internal/provider/incident_catalog_type_attribute_resource.go

```diff
@@ -30,6 +30,24 @@ var (
 	_ resource.ResourceWithImportState    = &IncidentCatalogTypeAttributeResource{}
 )
 
+// isSchemaOnlyMode returns true if the mode indicates that Terraform manages only
+// the schema (attribute definition), not the values. This includes:
+// - dashboard: user sets schema_only=true in Terraform
+// - internal: system changes mode when attribute is used as team's escalation path
+// - external: values managed by external integrations
+// - dynamic: values dynamically computed.
+func isSchemaOnlyMode(mode client.CatalogTypeAttributeV3Mode) bool {
+	switch mode {
+	case client.CatalogTypeAttributeV3ModeDashboard,
+		client.CatalogTypeAttributeV3ModeInternal,
+		client.CatalogTypeAttributeV3ModeExternal,
+		client.CatalogTypeAttributeV3ModeDynamic:
+		return true
+	default:
```

Comment:
I'm not sure if it's worth being explicit here and having `client.CatalogTypeAttributeV3ModeAPI`.